evm-ds/src/precompiles/scilla_call.rs

```diff
@@ -0,0 +1,132 @@
+use evm::backend::Backend;
+use evm::executor::stack::{PrecompileFailure, PrecompileOutput, PrecompileOutputType};
+use evm::{Context, ExitError};
+use std::borrow::Cow;
+
+use crate::precompiles::scilla_common::get_contract_addr_and_name;
+use crate::precompiles::scilla_common::substitute_scilla_type_with_sol;
+use ethabi::decode;
+use ethabi::param_type::ParamType;
+use ethabi::token::Token;
+use hex::ToHex;
+use serde_json::{json, Value};
+
+const BASE_COST: u64 = 15;
+const PER_BYTE_COST: u64 = 3;
```

Comment:
Leave a @todo here saying that we should move these to the config file. How did you arrive at them?